packages/ckeditor5-engine/src/view/treewalker.ts

```diff
@@ -159,6 +159,27 @@ export default class TreeWalker implements IterableIterator<TreeWalkerValue> {
 		}
 	}
 
+	/**
+	 * Moves treewalker {@link #position} to provided `position`. Treewalker will
+	 * continue traversing from that position.
+	 *
+	 * If the provided position is before the start boundary, the position will be
```

Comment:
```suggestion
	 * continue traversing from that position.
	 *
	 * Note: in contrary to {@link ~TreeWalker#skip}, this method does not iterate over the nodes along the way.
	 * It simply sets the current tree walker position to a new one.
	 * From the performance standpoint, it is better to use {@link ~TreeWalker#jumpTo} rather than {@link ~TreeWalker#skip}.
	 *
	 * If the provided position is before the start boundary, the position will be
```